### PR TITLE
add jade + bootstrap 3 generator

### PR DESCRIPTION
**Pull request by EranGoldman**

### CODE REVIEW

**Comment by IdanCo** [comment]
Looks great! I just think you forgot to do `git add` to the `jade-bootstrap3.transformer.ts` :-)

File: src/app/transformers/jade-bootstrap3.transformer.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import {Transformer} from "./transformer";
+
+export class JadeBootstrap3Transformer implements Transformer {
+  name = 'Jade & Bootstrap 3';
```
Comment (IdanCo) [inline src/app/transformers/jade-bootstrap3.transformer.ts:4]: Maybe
`name = 'Pug (FKA Jade) & Bootstrap 3'`
?

File: src/app/transformers/jade-bootstrap3.transformer.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  getFormHeader(): string {
+    return `
+.row
+  .col-md-12.col-sm-6.col-xs-12
+    .panel.panel-default
+      .panel-heading.clearfix
+        i.icon-calendar
+        h3.panel-title Title
+      .panel-body
```
Comment (IdanCo) [inline src/app/transformers/jade-bootstrap3.transformer.ts:19]: This is a very specific implementation (grid+responsive classes + panels + icons).
I think its better keeping it minimal, so people can do their own adaptation.
Perhaps use [HTML2Jade](http://html2jade.org/) to convert the current BS3+Angular template?

**Review by IdanCo — CHANGES_REQUESTED** [review]
Great stuff!!

See comments, and i think you can delete `package-lock.json` file from the PR.

Thank you!!

File: src/app/transformers/jade-bootstrap3.transformer.ts (hunk 2 above)
Comment (EranGoldman) [reply src/app/transformers/jade-bootstrap3.transformer.ts:19]: I don't use angular so I tried to keep it general, the icons are from the default BS3 and the grid is quite standard for inline forms (2, 10) 
I thought about adding more options to the object but I think it'll complicate staff without adding value
I can add a "Pug (FKA Jade) & Angular & Bootstrap 3" template

**Comment by IdanCo** [comment]
I really appreciate your effort, but after some deliberations I believe adding a transformer for jade is not very useful, since other services ([like this one](http://html2jade.org/)) can already convert any html to jade.

Also, without the angular directives I'm not sure if it is still useful.

Anyway you've done a great job, and if you or anyone else think differently feel free to reopen this PR or open an issue.

Thanks!

**Pull request closed without merging**